.github/workflows/assign-on-comment.yml

```diff
@@ -0,0 +1,35 @@
+name: Auto-assign issue on comment

```

Comment:
May we add here a commentary like `The code was heavily inspired by the https://github.com/delta-io/delta-rs/blob/main/.github/workflows/issue_comments.yml`?